iceoryx_hoofs/test/moduletests/test_cxx_helplets.cpp

```diff
@@ -189,4 +194,244 @@ TYPED_TEST(Helplets_test_isPowerOfTwo, MaxValueForTypeIsNotPowerOfTwo)
 {
     EXPECT_FALSE(isPowerOfTwo(static_cast<typename TestFixture::CurrentType>(TestFixture::MAX)));
 }
+
+TEST(Helplets_test_isValidFileName, EmptyNameIsInvalid)
+{
+    EXPECT_FALSE(isValidFileName(string<FILE_PATH_LENGTH>("")));
+}
+
+TEST(Helplets_test_isValidFileName, RelativePathComponentsAreInvalid)
+{
+    EXPECT_FALSE(isValidFileName(string<FILE_PATH_LENGTH>(".")));
+    EXPECT_FALSE(isValidFileName(string<FILE_PATH_LENGTH>("..")));
+}
+
+TEST(Helplets_test_isValidFileName, FileNameWithValidSymbolsAndDotsAreValid)
+{
+    EXPECT_TRUE(isValidFileName(string<FILE_PATH_LENGTH>("fuu.")));
+    EXPECT_TRUE(isValidFileName(string<FILE_PATH_LENGTH>("fuu..")));
+    EXPECT_TRUE(isValidFileName(string<FILE_PATH_LENGTH>("..bla")));
+    EXPECT_TRUE(isValidFileName(string<FILE_PATH_LENGTH>(".blubb")));
+    EXPECT_TRUE(isValidFileName(string<FILE_PATH_LENGTH>("scna..bla")));
+    EXPECT_TRUE(isValidFileName(string<FILE_PATH_LENGTH>("scna.blubb")));
+}
+
+TEST(Helplets_test_isValidFileName, ValidLetterCombinationsAreValid)
+{
+    std::array<std::string, 3> combinations;
+
+    for (uint32_t i = 0U; i <= 255U; ++i)
+    {
+        if ((65 <= i && i <= 90) ||  // A-Z
```

Comment:
please use constexpr like `constexpr uint32_t UPPER_CASE_A {static_cast<uint32_t>('A')}` and remove the comments